src/Umbraco.Tests/Testing/TestOptionAttributeBase.cs

```diff
@@ -24,13 +26,69 @@ public static TOptions GetFixtureOptions<TOptions>(Type type)
         {
             return Get<TOptions>(type, null);
         }
+        /// <summary>
+        /// Find the class name and method name that has the attribute
+        /// </summary>
+        /// <param name="callerName">Out Caller Method Name</param>
+        /// <returns>Caller Class Name</returns>
+        public static string GetBenchmarkTestCaller(out string callerName)
```

Comment:
This is pretty nasty, do we really need this? Surely we can achieve what is needed in a nicer way?